cuda/solver/common_trs_kernels.cuh

```diff
@@ -120,12 +120,13 @@ struct CudaSolveStruct : gko::solver::SolveStruct {
         const auto rows = matrix->get_size()[0];
         // workaround suggested by NVIDIA engineers: for some reason
         // cusparse needs non-nullptr input vectors even for analysis
+        // also make sure they are aligned by 16 bytes
```

Comment:
16bytes or 8 bytes?

Reply:
double complex needs to be aligned by 16 bytes, since `thrust::complex<double>` has higher alignment requirements to enable vectorized loads/stores

Reply:
but `0xDEAD0` only use more than two bytes, does it only has four byte?

Reply:
this is a pointer, the 0 at the least significant digit makes it divisible by 16